### PR TITLE
refactor layout

### PR DESCRIPTION
**Pull request by mkozhukharenko**

### CODE REVIEW

File: src/components/layout.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+import Header from '../components/header/Header';
+import Footer from '../components/footer/Footer';
+import '../styles/index.scss';
+import '../styles/colors.scss';
+
+const Layout = ({ children }) => (
+  <main className="page-main">
+    <Header />
+    <div className="page-main__content">
```
Comment (VasiliyPolyakov) [inline src/components/layout.js]: You changed className (page-main__content instead content__wrapper), but you didn't rename this class in indes.scss.

File: src/styles/index.scss [hunk 2]
```diff
@@ -24,21 +24,7 @@
...
-  @include fontPrimary(1);
-  text-transform: uppercase;
-  @include mq(tablet) {
-    @include fontPrimary(3);
-  }
-}
-
+// -- - --- - Do we need this styles?
 .content {
   min-height: 100%;
```
Comment (VasiliyPolyakov) [inline src/styles/index.scss]: i will delete it

File: src/styles/index.scss [hunk 3]
```diff
@@ -52,3 +38,28 @@ h1 {
     min-height: calc(100vh - 300px);
```
Comment (VasiliyPolyakov) [inline src/styles/index.scss]: Can you rename pls content-wrapper and content__wrapper to page-main__content

File: src/components/layout.js (hunk 1 above)
Comment (mkozhukharenko) [reply src/components/layout.js]: done

File: src/styles/index.scss (hunk 3 above)
Comment (mkozhukharenko) [reply src/styles/index.scss]: done

**Pull request merged**